packages/grafana-ui/src/components/Monaco/CodeEditor.tsx

```diff
@@ -19,14 +32,17 @@ class UnthemedCodeEditor extends React.PureComponent<Props> {
 
   componentDidUpdate(oldProps: Props) {
     const { getSuggestions, language } = this.props;
-    if (getSuggestions) {
-      // Language changed
-      if (language !== oldProps.language) {
-        if (this.completionCancel) {
-          this.completionCancel.dispose();
-        }
-        this.completionCancel = registerSuggestions(language, getSuggestions);
+
+    if (language !== oldProps.language && getSuggestions) {
+      if (this.completionCancel) {
+        this.completionCancel.dispose();
       }
+
+      if (!this.monaco) {
+        throw new Error('Monaco instance not loaded yet');
```

Comment:
what does this end up doing?  should we show something in react instead of throwing an error?

Reply:
Yeah, we should probably just log a warning or something instead?

the monaco instance is loaded async, so if the props change before monaco has loaded it'll fall into this.